### ISSUE TITLE
Superset homepage not working/up-to-date

### ISSUE THREAD

**Issue by villebro** [apache/incubator-superset#5097]
Make sure these boxes are checked before submitting your issue - thank you!

- [x] I have checked the superset logs for python stacktraces and included it here as text if any
- [x] I have reproduced the issue with at least the latest released version of superset
- [x] I have checked the issue tracker for the same issue and I haven't found one similar

### Expected results
https://superset.incubator.apache.org/ and https://superset.apache.org/ should correspond to the most recent production version of Superset (0.25.5 as of writing)

### Actual results
The homepage is not up to date, and some links don't work, e.g. https://superset.incubator.apache.org/security.html

**Comment by mistercrunch** [comment]
It'd be great to automate the publication process or formalize it as part of the release process.

**Comment by mistercrunch** [comment]
Agreed about keeping docs around for multiple versions. Out-of-the-box apache.org doesn't offer that, we'd have to probably use readthedocs on top of that which is somewhat easy.

Note that for API docs it's super important to have versioned docs but for Superset I feel like it's less critical. Definitely nice to have though.

**Comment by villebro** [comment]
I fully agree, right now valuable core developer time is best spent implementing awesome new functional features :)
